apps/Gruntfile.js

```diff
@@ -318,6 +318,8 @@ module.exports = function (grunt) {
         {pattern: 'lib/**/*', watched: false, included: false, nocache: true},
         {pattern: 'build/**/*', watched: false, included: false, nocache: true},
         {pattern: 'static/**/*', watched: false, included: false, nocache: true},
+        {pattern: 'lib/p5play/p5.js'},
+        {pattern: 'lib/p5play/p5.play.js'},
```

Comment:
@pcardune I'm pretty sure we did this correctly, can you confirm?  p5.js and p5.play.js absolutely insist on being included as script tags so they can load stuff onto window (they are not requirejs-compatible) so we put them here to include them in tests ahead of any of the actual test files.

It's not great that this means they'll be included for any/every test run of any kind, but I didn't see a better way to do this and use them in our gamelab unit tests.  I also doubt they'll slow the build down very much, as they're both prebuilt libraries, leaf nodes in the dependency tree.

Thoughts?